### PR TITLE
Add Fleet and Elastic Agent 8.3.0 release notes

### PR DESCRIPTION
**Pull request by dedemorton**
Changes:

- Adds template for 8.3 release notes.
- Removes release notes for old versions (7.13-8.2)
- Adds text to FAQ to replace the tagged region include.
**Reviewers:** Please indicate whether we need to keep the text in the FAQ or remove the section.

Remaining TODOs:
- [x] Add Fleet-related release notes when Kibana relnotes are available (copy/paste, modify to make links work).
- [x] Check for any new commits made after [BC4](https://staging.elastic.co/8.3.0-fcbaa721/summary-8.3.0.html#elastic-agent).
- [x] Get content reviewed by Fleet and Elastic Agent teams and mark this PR ready for review.
- [x] During the review, ask reviewers if there are any known problems or deprecations that need to be documented. (There's a template at the end of the document where you can copy/paste those sections back into the main document, if needed).
- [x] Remove the text beginning with `//` that appears at the end of some items to explain why the item was added or moved.

### CODE REVIEW

**Review by jlind23 — APPROVED** [review]

File: docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+=== New features
+
+The 8.3.0 release adds the following new and notable features.
+
+{fleet}::
+* Changes to agent upgrade modal to allow for rolling upgrades {kib-pull}132421[#132421]
+
+{agent}::
+* Add ability to <<filter-agent-list-by-tags,set tags>> during {agent}
+installation and enrollment. {agent-issue}149[#149] {agent-pull}336[#336] //Wasn't in changelog but seems important
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc:40]:
```suggestion
installation and enrollment. {agent-issue}149[#149] {agent-pull}336[#336]
```

File: docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+* Adds APM instrumentation for package install process {kib-pull}131223[#131223]
+* Adds "Label" column + filter to Agent list table {kib-pull}131070[#131070]
+* Adds `cache-control` headers to key `/epm` endpoints in Fleet API {kib-pull}130921[#130921]
+* Optimize package installation performance, phase 1 {kib-pull}130906[#130906]
+* Adds experimental features (feature flags) config to fleet plugin {kib-pull}130253[#130253]
+* Adds redesigned Fleet Server flyout {kib-pull}127786[#127786]
+
+{agent}::
+* Bump `node.js` version for Heartbeat/synthetics to 16.15.0.
+{agent-pull}446[#446] //Moved from New features section
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc:67]:
```suggestion
{agent-pull}446[#446]
```

File: docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+* Add extra k8s resources in `clusterRole` to better filter objects in
+dashboards and visualizations. {agent-pull}424[#424] //Wasn't in changelog but
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc:69]:
```suggestion
dashboards and visualizations. {agent-pull}424[#424]
```

File: docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+seems worth documenting.
+* Collect Endpoint Security logs on the `elastic-agent diagnostics collect`
+command. {agent-issue}105[#105] {agent-pull}242[#242] //Wasn't in changelog but
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc:72]:
```suggestion
command. {agent-issue}105[#105] {agent-pull}242[#242]
```

File: docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+seems worth documenting.
+* Add @metadata.input_id and @metadata.stream_id when applying the inject stream processor {agent-issue}527[#527] // added from recent Agent changelog edits
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc:74]:
```suggestion
* Add @metadata.input_id and @metadata.stream_id when applying the inject stream processor {agent-issue}527[#527]
```

File: docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+* Add liveness endpoint, allow fleet-gateway component to report degraded state, add update time and messages to status output {agent-issue}390[#390] {agent-pull}569[#569] // added from recent Agent changelog edits
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc:75]:
```suggestion
* Add liveness endpoint, allow fleet-gateway component to report degraded state, add update time and messages to status output {agent-issue}390[#390] {agent-pull}569[#569]
```

File: docs/en/ingest-management/troubleshooting/faq.asciidoc [hunk 7]
```diff
@@ -217,12 +217,22 @@ same way as other integrations. Try it out! For more information, see the
 presented by {kib}. The agent then provides an API key as an authentication
 token, which {kib} validates.
 
+//REVIEWERS: Is the following statement still true? It was supposed to be
+//fixed after 7.14. I've copied the statement directly into this document
+//so you can review it and decide whether to remove or include it going forward.
+
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting/faq.asciidoc:223]:
```suggestion
```

**Review by jlind23 — APPROVED** [review]

File: docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+* Bump `node.js` version for Heartbeat/synthetics to 16.15.0.
+{agent-pull}446[#446]
+* Add extra k8s resources in `clusterRole` to better filter objects in
+dashboards and visualizations. {agent-pull}424[#424]
+seems worth documenting.
+* Collect Endpoint Security logs on the `elastic-agent diagnostics collect`
+command. {agent-issue}105[#105] {agent-pull}242[#242]
+seems worth documenting.
+* Add @metadata.input_id and @metadata.stream_id when applying the inject stream processor {agent-issue}527[#527]
+* Add liveness endpoint, allow fleet-gateway component to report degraded state, add update time and messages to status output {agent-issue}390[#390] {agent-pull}569[#569]
```
[the remark this answers is not in the document]
Comment (ollyhowell) [reply docs/en/ingest-management/release-notes/release-notes-8.3.asciidoc]: Thanks, removed these two lines in last commit

**Review by jen-huang — APPROVED** [review]
Fleet LGTM

**Pull request merged**